src/main/java/edu/harvard/iq/dataverse/dataverse/messages/TextMessagePage.java

```diff
@@ -0,0 +1,11 @@
+package edu.harvard.iq.dataverse.dataverse.messages;
+
+import javax.faces.view.ViewScoped;
+import javax.inject.Named;
+import java.io.Serializable;
+
+@ViewScoped
+@Named
+public class TextMessagePage implements Serializable {
+
```

Comment:
check on init that user is dataverse admin and in dataverse we can allow text messages